### PR TITLE
fix: firm identity integration

### PR DESCRIPTION
**Pull request by sugumar-mani**

### CODE REVIEW

**Review by sugumar-mani — COMMENTED** [review]
LGTM (without unit test and FE console error fixes)

File: src/components/secureRoute/__tests__/index.test.jsx [hunk 1]
```diff
@@ -11,19 +10,20 @@ jest.mock('react-router-dom', () => ({
...
 describe('SecureRoute', () => {
   it('should render', () => {
     useParams.mockReturnValue({projectId: '1'});
     useNavigate.mockReturnValue(jest.fn());
-    useOktaAuth.mockReturnValue({
-      authState: {
-        isAuthenticated: true,
-      }
-    });
+    // useOktaAuth.mockReturnValue({
```
Comment (onegoel) [inline src/components/secureRoute/__tests__/index.test.jsx:22]: is this required?

Comment (sugumar-mani) [reply src/components/secureRoute/__tests__/index.test.jsx:22]: can be removed while fixing unit tests

**Review by onegoel — APPROVED** [review]
LGTM

**Pull request merged**